lib/mongoose/index.js

```diff
@@ -39,6 +39,8 @@ const MongoosePlugin = {
 
         Promise.all(queue).then(() => {
             next();
+        }).catch((e) => {
+            console.log(e);
```

Comment:
Please add a tag before to indicate this error is from mongoose plugin. Like `console.log('[mongoose'], e)`